### PR TITLE
fix(docs-infra): fix aio homepage breaking on certain widths

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
the aio homepage has a fixed width for its hero content,
that is not handling well certain window widths, fix that by removing
the fixed width and tweaking related media queries

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

The aio homepage does handle well certain window widths:

Peek 2021-11-09 22-05

## What is the new behavior?

The issue does not happen anymore and all window width's are handled

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by dario-piotrowicz** [comment]
@gkalpak the second commit (`fix(docs-infra): remove negative margin`) is unrelated.... I just couldn't resist as I find negative margins hacky... if you don't agree with that change please let me know and I can drop the commit no problem :sweat_smile:

File: aio/src/styles/1-layouts/footer/_footer.scss [hunk 1]
```diff
@@ -82,7 +82,7 @@ footer {
         }
       }
 
-      @media (max-width: 480px) {
+      @media (max-width: 40rem) {
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/footer/_footer.scss:85]: OOC, why this change?

**Review by gkalpak — APPROVED** [review]
One question. Otherwise lgtm :+1:
Thx, @dario-piotrowicz!

File: aio/src/styles/1-layouts/footer/_footer.scss (hunk 1 above)
Comment (dario-piotrowicz) [reply aio/src/styles/1-layouts/footer/_footer.scss:85]: it's to avoid the issue of the page breaking because of the footer on different font-sizes

as you can see, with 480px the page can still break:
480px

while with 40rem it never does:
40rem


But I actually overshoot it quite a bit, seems like `32rem` there is actually more than enough... :sweat: sorry I'll make the change :slightly_smiling_face:

Comment (gkalpak) [reply aio/src/styles/1-layouts/footer/_footer.scss]: Thx for the explanation (and the fix) :smiley:

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit d8ae34aea8850002ca9c300ea6cf99d14535a4ec.
